community/installation-guides/panel/debian10.md

```diff
@@ -23,13 +23,13 @@ systemctl start mariadb
 systemctl enable mariadb
 ```
 
-### PHP 7.4
+### PHP 8.0
 ```bash
 ## Get apt updates
 apt update
 
 ## Install PHP 7.4
```

Comment:
This comment should be changed to PHP 8.0.

I am fairly sure Debian 10 will not have PHP 8.0 packages out of the box. Doesn't it require you to add a new source, for example, from sury? I think only difference between official documentation and Debian ones are the PPA repositories